src/Introduction.scss

```diff
@@ -32,17 +33,32 @@ $spacer : 1rem;
     background: linear-gradient(120deg,rgba(0,0,0,50%),hsl(46, 65%, 52%))
 }
 .intro-article {
-    height: 16rem;
+    height: auto;
     border-radius: 12px;
     padding: 1rem;
 }
-.pic {
+.img-container {
+    position: relative;
+    width: 20rem;
     height: 30rem;
+    overflow: hidden;
+    margin-right: 3rem;
+}
+.responsive-image {
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
```

Comment:
does this even do anything with position absolute?

Reply:
No it does not. I would love to have a talk about position absolute/relative with you. I'm half way there to get it.